libs/blocks/merch-card/merch-card.css

```diff
@@ -0,0 +1,375 @@
+  position: absolute;
+  top: 16px;
+  right: 0;
+  font-size: 14px;
+  font-weight: 500;
+  max-width: 150px;
+  line-height: 16px;
+  text-align: center;
+  padding: 8px 11px;
+  border-radius: 5px 0 0 5px;
+}
+
+.merch-card .consonant-ProductCard-cell,
+.merch-card .consonant-SegmentBlade-cell,
+.merch-card .consonant-SpecialOffers-cell,
+.merch-card .consonant-PlansCard-cell {
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.merch-card p {
+  font-size: 14px;
+  font-weight: 400;
+  line-height: 1.5;
+  margin-bottom: 8px;
+  margin-top: 0;
+  color: #000;
+}
+
+.merch-card .consonant-SpecialOffers-inner,
+.merch-card .consonant-PlansCard-inner {
+  justify-content: center;
+}
+
+.merch-card.special-offers .consonant-SpecialOffers-description p,
+.merch-card.special-offers .consonant-PlansCard-description p {
+  font-size: 14px;
+}
+
+.merch-card.special-offers .consonant-CardFooter {
+  margin-top: 0;
+  padding-top: 0;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.merch-card.special-offers h4 {
+  font-size: 12px;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+}
+
+.merch-card .consonant-CardFooter .standard-wrapper,
+.merch-card .consonant-CardFooter .checkbox-container {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.merch-card .consonant-CardFooter .checkbox-container {
+  justify-content: flex-start;
+  margin-bottom: 12px;
+  cursor: pointer;
+}
+
+.merch-card .consonant-CardFooter .standard-wrapper {
+  height: 32px;
+  white-space: nowrap;
+}
+
+.merch-card .secure-transaction-wrapper {
+  display: flex;
+  align-items: center;
+  white-space: nowrap;
+  flex-shrink: 1;
+}
+
+.merch-card .secure-transaction-label {
+  font-size: 12px;
+  color: #505050;
+  margin-left: 12px;
+}
+
+.merch-card .secure-transaction-icon {
+  display: inline-block;
+}
+
+.merch-card .secure-transaction-icon-img {
+  display: block;
+  height: 15px;
+  width: 12px;
+}
+
+
+.merch-card .consonant-CardFooter .checkbox-container .checkbox-label {
+  font-size: 12px;
+  color: #505050;
+  white-space: nowrap;
+}
+
+.merch-card .secure-transaction-label,
+.merch-card .consonant-CardFooter .checkbox-container .checkbox-label {
+  flex-grow: 1;
+  margin-left: 12px;
+}
+
+.merch-card .secure-transaction-icon,
+.merch-card .checkbox-container .checkmark {
+  flex-shrink: 0;
+}
+
+.merch-card .consonant-CardFooter .checkbox-container input[type="checkbox"] {
+  display: none;
+}
+
+.merch-card .consonant-CardFooter .checkbox-container .checkmark {
+  position: relative;
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  border: 2px solid #757575;
+  background: #fff;
+  border-radius: 2px;
+  cursor: pointer;
+}
+
+.merch-card .consonant-CardFooter .checkbox-container input[type="checkbox"]:checked + .checkmark {
+  background-color: #2680eb;
+  background-image: url('./img/check-mark-small.svg');
+  border-color: #2680eb;
+}
+
+@media only screen and (min-width: 680px) {
```

Comment:
Our typical media queries are `600` && `1200` w/ occasional `900` and `1440`. 

Where are these values coming from? 
We should aim to keep consistent viewports on all blocks.

Reply:
Also we generally keep them in order from lowest to greatest where possible at the end of the file. 
```
>600...

>1200...

>1440...
```